Automattic/pocket-casts-ios
# [Winback]  Apply Apple API for subscription cancellation

| 📘 Part of: #2435
|:---:|

Fixes #2444 
Fixes #2445 

This PR uses the `StorerKit.AppStore` API to present the subscriptions panel and handle those

## To test

- CI must be 🟢 
- Make sure you enabled `winback` FF
- Use a real device with a sandbox account
- Use a Plus/Patron account or a free account and purchase a Plan
- Got to your account, Cancel Subscription
- Tap on Continue to Cancellation
- On the new screen tap on Yes, Cancel my Subscription 
- You should see the Apple screen to manage the sunscription
- Tap on cancel the subscriptions
- Dismiss the view
- Now, the API to check the receipt may take some times. I need to investigate if we can get the new status quickly. If the state doesn't reload try to force reload your account till the new status updates

## Checklist
